src/main/java/com/budgetapp/controllers/CategoryController.java

```diff
@@ -7,20 +7,22 @@
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
+@RequestMapping(path = "/api/categories")
 public class CategoryController {
 
     @Autowired
     private CategoryService categoryService;
 
-    @PostMapping("/addCategory")
+    @PostMapping()
     public void addCategory(@RequestBody Category category) {
         categoryService.addCategory(category);
     }
```

Comment:
```suggestion
    @PostMapping(
         consumes = MediaType.APPLICATION_JSON_VALUE,
         produces = MediaType.APPLICATION_JSON_VALUE
    )
    public ResponseEntity<Category> addCategory(@RequestBody final Category category) {
        final var createdCategory = categoryService.addCategory(category);
        return ResponseEntity
                .status(HttpStatus.CREATED)
                .body(createdCategory );
    }
```
Please make sure all your controller endpoints follow the standard of returning a `ResponseEntity` object with the correct statuses and bodies based on what they are doing. Check the [HttpStatus Codes](https://www.semrush.com/blog/http-status-codes/?kw=&cmp=Africa_SRCH_DSA_Blog_EN&label=dsa_pagefeed&Network=g&Device=c&utm_content=622528804255&kwid=dsa-1754723155433&cmpid=18364843126&agpid=141057021083&BU=Core&extid=60114153143&adpos=&gclid=CjwKCAjwoIqhBhAGEiwArXT7K5dcw8D9JDfkxvB-rWrEhk038xRrqZb4koXr7eeEE14ZuOZcxYCNvhoCi-oQAvD_BwE) link.

Reply:
Please make sure you apply the above suggestion to all endpoints.